.github/workflows/ci.yml

```diff
@@ -13,7 +13,7 @@ on: # yamllint disable-line rule:truthy rule:comments
   pull_request: ~
 
 env:
-  PLUGIN_NAME: "nautobot-plugin-floor-plan"
+  PLUGIN_NAME: "nautobot-app-floor-plan"
```

Comment:
What would be required to change `PLUGIN_NAME` to `APP_NAME`?

Reply:
On the surface I don't think it would matter looking at the CI file